doc/adr/0007-generate-qr-code-on-client

```diff
@@ -0,0 +1,25 @@
+# 1. Record architecture decisions
+
+Date: 2022-06-30
+
+## Status
+
+Accepted
+
+## Context
+
+QR Code is embedded in the eService page and it is used to allow user to start identification process via the UseID app camera which later. The QR Code contains `tcTokenURL`, `widgetSessionId` and `encryptionKey`. For this decision record, [ADR 0006](LINK TO 0006) 
+```
+eid://127.0.0.1:24727/eID-Client?tcTokenURL={{tcTokenURL}}&widgetSessionId={{widgetSessionId}}&mobile={{isMobileDevice}}#{{encryptionKey}}
+```
+The current QR Code generation is handled in the server `/v1/qrcode/{imageSize}`
```

Comment:
Since we have no ADR about that, I'd just not mention it. We still should add some context, why there is a problem:
> The `encryptionKey`, which needs to be encoded in the QR code, must not leave the user's device, i.e. it must not be sent to a server.

And maybe add something about the `tcTokenURL`, that it contains information about the session, the server should (or must) not be aware of?